bindings/pyroot/pythonizations/test/CMakeLists.txt

```
@@ -26,8 +26,9 @@ ROOT_ADD_PYUNITTEST(pyroot_pyz_decorator pythonization_decorator.py)
 ROOT_ADD_PYUNITTEST(pyroot_pyz_pretty_printing pretty_printing.py)
 ROOT_ADD_PYUNITTEST(pyroot_pyz_array_interface array_interface.py PYTHON_DEPS numpy)
 
-# STL vector pythonizations
+# STL containers pythonizations
 ROOT_ADD_PYUNITTEST(pyroot_pyz_stl_vector stl_vector.py)
+ROOT_ADD_PYUNITTEST(pyroot_pyz_stl_set stl_set.py)
```

It looks like `ROOT.std.set` doesn't work on WIndows. So for the time being I propose to disable it:
```suggestion
if(NOT MSVC OR win_broken_tests)
  ROOT_ADD_PYUNITTEST(pyroot_pyz_stl_set stl_set.py)
endif()
```